xml/System.Net.Http/HttpClient.xml

```diff
@@ -730,8 +731,7 @@ The <paramref name="requestUri" /> is not an absolute URI.
 
 <see cref="P:System.Net.Http.HttpClient.BaseAddress" /> is not set.</exception>
         <exception cref="T:System.Net.Http.HttpRequestException">The request failed due to an underlying issue such as network connectivity, DNS failure, server certificate validation or timeout.</exception>
-        <exception cref="T:System.Threading.Tasks.TaskCanceledException">.NET Core and .NET 5 and later only: The request failed due to timeout.</exception>
-        <exception cref="T:System.OperationCanceledException">The cancellation token was canceled. This exception is stored into the returned task.</exception>
+        <exception cref="T:System.OperationCanceledException">.NET Core and .NET 5 and later only: The request failed due to timeout. This exception is stored into the returned task.</exception>
```

Comment:
Completely removing the next line (L734) seems a bit incorrect?
The resulting text implies that this exception is thrown only on .NET Core and .NET 5+, which is not true.

I guess here (and in all following similar occurrences) it should be something like this?

```suggestion
        <exception cref="T:System.OperationCanceledException">The cancellation token was canceled. This exception is stored into the returned task.
-or-
.NET Core and .NET 5 and later only: The request failed due to timeout.</exception>
```